### PR TITLE
Replaced iconRight with iconPosition

### PR DESCRIPTION
**Pull request by hetanthakkar1**
Solved issue #2591

### CODE REVIEW

**Comment by pranshuchittora** [comment]
I think this is not required.

**Comment by hetanthakkar1** [comment]
@pranshuchittora  Would you please tell me what i did wrong

**Comment by flyingcircle** [comment]
I agree with @pranshuchittora the code isn't very clean on this. Listing consecutive if statements is a bad design.

**Comment by hetanthakkar1** [comment]
@pranshuchittora @flyingcircle Can you please review it!

File: src/buttons/Button.tsx [hunk 1]
```diff
@@ -42,7 +42,7 @@ export type ButtonProps = TouchableOpacityProps &
     containerStyle?: StyleProp<ViewStyle>;
     icon?: IconNode;
     iconContainerStyle?: StyleProp<ViewStyle>;
-    iconRight?: boolean;
+    iconPosition?: string;
```
Comment (arpitBhalla) [inline src/buttons/Button.tsx:45]: iconPosition?:  "left" | "right" | "top" | "bottom"

**Comment by hetanthakkar1** [comment]
@flyingcircle @pranshuchittora I have made the necessary changes. Please review

File: src/buttons/Button.tsx [hunk 2]
```diff
@@ -133,6 +135,21 @@ const Button: RneFunctionComponent<ButtonProps> = (props) => {
...
+      case 'bottom':
+        [styles.buttonOrientation, { flexDirection: 'column-reverse' }];
+        break;
+      case 'right':
+        [styles.buttonOrientation, { flexDirection: 'row-reverse' }];
+        break;
+      default:
+        [styles.buttonOrientation, { flexDirection: 'row' }];
+    }
+  };
```
Comment (arpitBhalla) [inline src/buttons/Button.tsx:152]: We can simply use object 
let iconStyle={top:"column",...other positions...}[iconPosition] || "column";

**Pull request closed without merging**